Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/ApayaTokenizationViewModel.swift

```diff
@@ -255,11 +255,14 @@ class ApayaTokenizationViewModel: PaymentMethodTokenizationViewModel {
             return
         }
 
-        guard let currencyStr = AppState.current.currency?.rawValue else {
-            let err = PrimerError.invalidSetting(name: "currency", value: nil, userInfo: ["file": #file,
-                                                                                          "class": "\(Self.self)",
-                                                                                          "function": #function,
-                                                                                          "line": "\(#line)"], diagnosticsId: UUID().uuidString)
+        guard let currencyStr = AppState.current.currency?.code else {
+            let err = PrimerError.invalidSetting(name: "currency",
+												 value: nil,
+												 userInfo: ["file": #file,
+															"class": "\(Self.self)",
+															"function": #function,
+															"line": "\(#line)"],
+												 diagnosticsId: UUID().uuidString)
```

Comment:
Formatting here too

Reply:
<img width="721" alt="Screenshot 2024-01-29 at 18 46 27" src="https://github.com/primer-io/primer-sdk-ios/assets/137198782/5d06d90d-db34-4391-87ce-2814b472dd3d">
this is how the code looks like in Xcode, not sure what is the deal here...